cobra/manipulation/add.py

```diff
@@ -0,0 +1,48 @@
+from cobra.core import Reaction
+
+
+def add_exchange(model, metabolite, direction='demand', bound=1000.0,
+                 prefix=None):
+    """Add an exchange reaction (demand or uptake) for a given metabolite.
+
+    Parameters
+    ----------
+    model : cobra.Model
+        The model to add the exchange reaction to.
+    metabolite : cobra.Metabolite
+    direction : string, optional
+        One of 'demand' for only import, 'sink' for only export or
+        'exchange' for reversible export or uptake.
+    bound : float, optional
+        Upper and lower bound for the exchange reaction. Demand gets
+        (0, bound), sink gets (-bound, 0) and exchange gets (-bound,
+        bound).
+    prefix : string, optional
+        The prefix to use for the reaction. The default is to use 'EX_' for
+        'exchange', 'DM_' for demand and 'SK_' for sink reactions.
+
+    Returns
+    -------
+    cobra.Reaction
+        The created exchange reaction.
+
+    """
+    exchange_types = {'demand': ('DM_', (0, bound)),
+                      'sink': ('SK_', (-bound, 0)),
```

Comment:
A sink is not the same as an export. Sink means the metabolite "disappears" inside of the compartment (for instance if there are unknown consuming reactions). An export means the metabolite is transported to the outside across the compartment boundary. Both look the same in stoichiometry but only the export would be considered in some algorithms (for instance modeling exchanges between several species), whereas the sink would not. So we should give a way to distinguish export and sink reactions by the prefix ("SK_" or "EX_"). 

My proposal: add "import" and "export" specifiers to the `exchange_types` and handle their prefix accordingly.

Thanks!